spec/requests/internal/profile_fields_spec.rb

```diff
@@ -0,0 +1,95 @@
+require "rails_helper"
+
+RSpec.describe "/internal/profile_fields", type: :request do
+  include ActiveJob::TestHelper
+  let(:user) { create(:user) }
+
+    before do
+      user.add_role(:super_admin)
+      sign_in user
+    end
+
+  describe "GET /internal/profile_fields" do
+    let(:profile_field) { create(:profile_field) }
+
+    it "renders successfully" do
+      get internal_profile_fields_path
+      expect(response).to be_successful
+    end
+
+    it "lists the profile fields" do
+      profile_field
```

Comment:
**suggestion (non-blocking):** You should be able to avoid doing this by using `let!` over `let` above. Or just don't use `let` at all, since this is the only place where you use it in this block anyway.

```suggestion
      profile_field = create(:profile_field)
```